app/scripts/components/common/layout-root/index.tsx

```diff
@@ -35,17 +43,58 @@ const PageBody = styled.div`
 `;
 
 function LayoutRoot(props: { children?: ReactNode }) {
+  const useConsentForm = checkEnvFlag(process.env.COOKIE_CONSENT_FORM);
+
+  const readCookie = (name) => {
```

Comment:
Any reason that this function is repeated in two files? 
https://github.com/NASA-IMPACT/veda-ui/pull/1127/files#diff-e42ad7e47cbcdcd4c87a29eea61c27201f72b653eb6484df90d4666f41682658R1
Since there is util file already, wouldn't it be better to consolidate these functions in util file?

Reply:
This was a change that fell through the cracks while rebasing. There should only be 1 instance of this function.